src/main/java/com/foodiefinder/cities/controller/CityController.java

```diff
@@ -0,0 +1,21 @@
+package com.foodiefinder.cities.controller;
+
+import com.foodiefinder.common.dto.ResponseDto;
+import com.foodiefinder.cities.service.CityService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cities")
```

Comment:
```suggestion
@RequestMapping("/api/cities")
```

팀 컨벤션에 따르면 api 경로는 /api로 시작해야합니다🥺